client/jetpack-cloud/sections/agency-dashboard/sites-overview/utils.ts

```diff
@@ -490,6 +490,11 @@ export const getProductSlugFromProductType = ( type: string ): string | undefine
 };
 
 export const availableNotificationDurations = [
+	{
+		time: 1,
+		label: translate( 'After 1 minute' ),
+		paid_tier: true,
```

Comment:
I think it is better to use `camelCase` here. Maybe name it `isPaidTier` or just `isPaid` since it is a boolean value? WDYT?

Reply:
I would concur, good eye @yashwin -- the only time I've seen `snake_case` used without hesitation is in handling data directly from the PHP-based back end. Everything originating inside the JavaScript codebase is camelCased.

Reply:
@yashwin good catch 👍 I didn't realize I was not using `camelCase`. Thanks for catching this. I have updated it and change to `isPaid`.